content/docs/en-us/getting-started.md

```diff
@@ -1,6 +1,244 @@
 ---
 title: Getting Started
+sideToc: false
 ---
-# Gettting Started
 
-Welcome to Rancher Dashboard.
\ No newline at end of file
+# Getting Started
+
+This short guide provides a basic introduction to the UI.
+
+If you have previously been using the Cluster Manager UI, check out the guide below to learn where you can access the equivalent functionality in the new UI.
+
+## Home
+
+When you first login to Rancher you'll be taken to the new Home page. There are a numebr of information cards presented. You can close these using the <i class="icon icon-close doc-icon"></i> icon. If you wish to bring back the hidden cards, open the page menu using the <i class="icon icon-actions doc-icon"></i>  icon in the top-right and choose 'Restore hidden cards'.
+
+You can change where you land when you login from the Home page and also from the Preferences page. To access User Preferences, click on the user avatar icon in the top right and choose 'Preferences' from the menu.
+### Top-level Menu
+
+The new slide-in menu provides quick access to your clusters and apps. It is accessible by clicking on the <svg class="doc-icon" xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0z" fill="none" /><path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z" /></svg> icon in the top-left.
```

Comment:
The svg colour doesn't respect dark mode (previous two icons using `<i` work fine)